zerorobot/cli/server.py

```diff
@@ -15,6 +15,13 @@
 telegram_logger = logging.getLogger('telegram_logger')
 telegram_logger.disabled = True
 
+if  j.core.state.configGetFromDict("myconfig", "backend") == "db":
```

Comment:
Why is is the in global scope ? this is going to fail every time you call any action from the CLI. are you sure this is the proper location for it ?

Reply:
What is more appropriate place? i think robot.py https://github.com/threefoldtech/0-robot/blob/development/zerorobot/robot/robot.py ? or even in the __init__ of robot package? 
What do you suggest?

Reply:
why can't it be in the start function itself. this needs to happens only when we start the robot no ?